addons/stock/static/src/widgets/counted_quantity_widget.js

```diff
@@ -35,8 +35,16 @@ export class CountedQuantityWidgetField extends FloatField {
 
     onKeydown(ev) {
         const hotkey = getActiveHotkey(ev);
-        if (["enter", "tab", "shift+tab"].includes(hotkey))
-            this.onInput(ev)
+        if (["enter", "tab", "shift+tab"].includes(hotkey)) {
+            let val = ev.target.value;
+            try {
+                val = this.parse(val);
+                this.props.update(val);
```

Comment:
No need to define `val` outside of the `try` block, except that it's perfect :+1: 
```suggestion
            try {
                const val = this.parse(ev.target.value);
                this.props.update(val);
```

Reply:
Thanks for the suggestion. I applied the change.